downloader_app/run_celery_downloader.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+
+import os
+import sys
+
+import pandas as pd
+
+from downloader_app import shapefile_module as shpm
+from downloader_app.tasks import add, download_source
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+WORK_DIR = os.path.join(BASE_DIR, "downloader_app")
+sys.path.insert(0, WORK_DIR)
+
+
+# Set the filepath and load in a shapefile.
+SHP_PATH = os.path.join(
+    WORK_DIR, "shape_region/RJ_Mun97_region/RJ_Mun97_region.shp"
```

Comment:
it shouldn't be hard-coded ... if you have plans to do it in another PR, add a `TODO` comment here.

Reply:
I agree. By the way, we should not be using any external shapefiles since we have the geojson for every municipality in the database.